tests/dragonfly/acl_family_test.py

```diff
@@ -220,27 +220,23 @@ async def test_acl_cat_commands_multi_exec_squash(df_factory):
     await client.close()
 
 
-@pytest.mark.skip("Skip because it fails on arm release")
 @pytest.mark.asyncio
 async def test_acl_deluser(df_server):
     client = aioredis.Redis(port=df_server.port)
 
-    res = await client.execute_command("ACL SETUSER george ON >pass +@transaction +@string")
-    assert res == b"OK"
-
-    res = await client.execute_command("AUTH george pass")
-    assert res == b"OK"
+    assert await client.execute_command("ACL SETUSER george ON >pass +set ~*") == b"OK"
+    assert await client.execute_command("AUTH george pass") == b"OK"
 
-    await client.execute_command("MULTI")
```

Comment:
This should `not` be removed. It was the whole point of the test, to `delete while we are in multi` and that is the reason it was failing. 

Plz bring `MULTI/EXEC` back.